lib/developer_portal/lib/liquid/forms/spam_protected.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Liquid
+  module Forms
+    class SpamProtected < Forms::Create
+      def initialize(context, _object_name, html_attributes = {})
+        # FIXME: object_name should be removed and the template password/new fixed. We're
+        # accidentally passing a bracket '{'
+        super(context, 'site_account', html_attributes)
```

Comment:
I do not understand this one, why `'site_account'`?  Is it a drop or the actual provider account?

Reply:
I think that's because of how `Liquid::Forms::Base#object` works.

https://github.com/3scale/porta/blob/5a719dc90fc3b7a277cb64755c1b4868a1279a54/lib/developer_portal/lib/liquid/forms/base.rb#L29

Reply:
OK so the form has `object` which how it builds the class names/attributes values
And it gets it from a drop which gets back the real `model`

Thanks for enlightening me, then the test needs to be fixed IMO